### ISSUE TITLE
Catch NoneType within parse_service_key check

### ISSUE THREAD

**Issue by hiddenillusion** [tomchop/volatility-autoruns#10]
Traceback:

```
Volatility Foundation Volatility Framework 2.5
Traceback (most recent call last):
  File "/opt/tools/volatility/vol.py", line 192, in <module>
    main()
  File "/opt/tools/volatility/vol.py", line 183, in main
    command.execute()
  File "/opt/tools/volatility/volatility/commands.py", line 119, in execute
    data = self.calculate()
  File "/other_plugins/community/ThomasChopitea/autoruns.py", line 519, in calculate
    self.services = self.get_services()
  File "/other_plugins/community/ThomasChopitea/autoruns.py", line 309, in get_services
    service = self.parse_service_key(service)
  File "/other_plugins/community/ThomasChopitea/autoruns.py", line 287, in parse_service_key
    entry = entry.replace('\x00', '')
AttributeError: 'NoneType' object has no attribute 'replace
```

In this instance, `entry` was empty.

**Comment by tomchop** [comment]
Try now? :)

**Comment by tomchop** [comment]
Closed in 095b268bf6dd828c054a5dc4cd41b957b49d028a
